client/src/components/Icons.tsx

```diff
@@ -204,3 +204,12 @@ export const ArrowIcon = (props: ArrowIconProps) => {
     </span>
   );
 };
+
+export const AddIcon = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
not happy to be adding this here but didn't wanna do it via component library at this moment